browser/templates/browser/investigator.html

```diff
@@ -79,6 +79,26 @@
         </div>
     </div>
     {% endwith %}
+
+    {% with form.institution as field %}
+    <div class="row">
+        <div class="col-xs-12">
+            {% with form.institution as field %}
+                <div class="form-group">
+                    {% for error in field.errors %}
+                    <div class="alert alert-warning">{{ error }}</div>
+                    {% endfor %}
+                    <label>{{ field.label }}</label>
+                    {{ form.institution_search|addcss:"form-control input-sm" }}
```

Comment:
I believe you still need to pass a csrf token in this form

Reply:
The csrf token has been given for the form at line number 38